frontend/src/views/DonationForm.vue

```diff
@@ -0,0 +1,245 @@
+            <v-menu
+              :style="{ height: '100%', width: '100%' }"
+              ref="menu"
+              v-model="menu"
+              :close-on-content-click="false"
+              :return-value.sync="date"
+            >
+              <div slot="activator">
+              </div>
+              <v-date-picker v-model="form.date" no-title scrollable>
+                <v-spacer></v-spacer>
+                <v-btn flat color="primary" @click="menu = false">Cancel</v-btn>
+                <v-btn flat color="primary" @click="$refs.menu.save(form.date)">OK</v-btn>
+              </v-date-picker>
+            </v-menu>
+          </v-flex>
+        </v-flex>
+        <v-flex xs12 sm7>
+          <v-flex xs4 sm4 md3 lg2 class="py-1 pl-1">
+            <v-combobox
+              class="currency"
+              outline
+              label="Currency"
+              v-model="form.sum.currency"
+              :items="currencies"
+            >
+            </v-combobox>
+          </v-flex>
+          <v-flex xs8 sm8 md9 lg10 class="py-1 pr-1">
+            <v-text-field class="amount" :rules="amountRules" outline label="Amount" v-model="form.sum.amount" />
+          </v-flex>
+        </v-flex>
+        <v-flex xs12 sm6 lg4 class="pa-1">
+          <v-text-field :rules="rules" outline label="Name" v-model="form.donor.name" />
+        </v-flex>
+        <v-flex xs12 sm6 lg4 class="pa-1">
+          <v-text-field
+            :rules="emailRules"
+            outline
+            label="Email Address"
+            v-model="form.donor.emailAddress"
+          />
+        </v-flex>
+        <v-flex xs12 lg4 class="d-flex" :style="{ flexDirection: 'row', flexWrap: 'wrap', padding: '4px 4px 4px 4px' }">
+          <vue-phone-number-input
+            :class="{ 'mb-2': true, 'phone-error': phoneError }"
+            @update="handleUpdatePhone"
+            :style="{ width: '100%' }"
+            no-flags
+            v-model="form.donor.phoneNumber"
+            required
+            no-validator-state
+            no-use-browser-locale
+          />
+          <div class="phone-error-message">{{ phoneError }}</div>
+        </v-flex>
+        <v-flex xs12 class="pa-1">
+          <v-text-field :rules="rules" outline label="Collected By" v-model="form.collectedBy" />
+        </v-flex>
+        <v-flex xs12 class="pa-1">
+          <v-textarea outline label="Comment" v-model="form.comment" />
+        </v-flex>
+        <v-btn :loading="isSubmitting" :disabled="isSubmitting" type="submit" color="success">Submit</v-btn>
+        <span
+          :style="{ color: submissionStatus === 'success' ? '#4caf50' : 'red'}"
+          class="d-flex align-center"
+        >
+          {{ submissionMessage }}
+        </span>
+      </v-layout>
+    </v-form>
+  </v-container>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+import firebase from "firebase/app";
+import fb from "../firebaseApp";
```

Comment:
This is not used anymore! We have eliminated this file at all while adding easy configuration for Firebase Hosting. So rebase your branch onto the latest `master` and correct here.